Makefile

```diff
@@ -159,15 +159,18 @@ crds:
 	cp -R bazel-genfiles/config/crds/* config/crds/
 	cp -R bazel-genfiles/config/rbac/* config/rbac/
 
-# TODO(vincepri): This should move to rebuild Bazel binaries once every
-# make target uses Bazel bins to run operations.
```

Comment:
Shouldn't these comments persist since `binaries-dev` is not yet updated to use bazel?

Reply:
I removed them because I think for the time being these dev binaries will stay on the GOPATH and should be rebuilt this way. Moving those to Bazel is optional and could be done later if there is real necessity.

Reply:
Actually, do we even need this step now? Or should it be updated to just call the `clusterctl` and `clusterawsadm` targets? Those targets already install the binaries into the ${GOPATH}/bin